openapi_python_client/templates/endpoint_macros.py.jinja

```diff
@@ -73,17 +73,17 @@ params = {k: v for k, v in params.items() if v is not UNSET and v is not None}
 
 {# The all the kwargs passed into an endpoint (and variants thereof)) #}
 {% macro arguments(endpoint) %}
+{# path parameters #}
+{% for parameter in endpoint.path_parameters %}
+{{ parameter.to_string() }},
+{% endfor %}
```

Comment:
Only problem with this is that the order of `endpoint.path_parameters` is not guaranteed. If the declaration order swaps around or there's a slight change in implementation (e.g. at some point we decide to alphabetize kwargs) then generated clients will break and potentially do so in a way that consumers don't catch.

If we want to allow positional args, we'll have to guarantee the order for them. For Path params we could use the order in which they appear in the path, which seems the most logical solution to me. It's going to require a bit more up front parsing work though.

Reply:
Thanks for the review! I will work on sort them in the order they appear in the path.

Reply:
Done in https://github.com/triaxtec/openapi-python-client/pull/429/commits/b9e8057c3dc96057ee609f7a61311a0f832b7b65